src/main/java/io/xpire/logic/commands/DeleteCommand.java

```diff
@@ -101,6 +101,10 @@ public CommandResult execute(Model model) throws CommandException, ParseExceptio
             assert this.quantity != null;
             Item newQuantityItem = reduceItemQuantity(targetItem, quantity);
             model.setItem(targetItem, newQuantityItem);
+            /* TODO: Transfer to To-Buy-List*/
+            if (Quantity.quantityIsZero(newQuantityItem.getQuantity())) {
+                model.deleteItem(targetItem);
```

Comment:
This isn't a permanent feature right? The item will be moved to the To-Buy List.

Reply:
yep! I've indicated in the comment on line that this is a todo.